resources/codegen_inputs/sbix.rs

```diff
@@ -0,0 +1,45 @@
+#![parse_module(read_fonts::tables::sbix)]
+
+/// The [sbix (Standard Bitmap Graphics)](https://docs.microsoft.com/en-us/typography/opentype/spec/sbix) table
+#[read_args(num_glyphs: u16)]
+#[tag = "sbix"]
+table Sbix {
+    /// Table version number — set to 1.
+    #[compile(1)]
+    version: u16,
+    /// Bit 0: Set to 1.
+    /// Bit 1: Draw outlines.
+    /// Bits 2 to 15: reserved (set to 0).
+    flags: u16,
```

Comment:
we might want to define a flags type for this?

Reply:
Good call. The unnamed bit 0 that must always be set to 1 seems problematic for the codegen flags type. Wdyt about wrapping in a record instead and implementing default and appropriate methods for the “render outlines bit” instead?

Reply:
I think an easier option would be to use flags, but use the `compile_with` annotation to provide a fn that ensures that bit one is always set when writing.

Reply:
Ah, will do!

Reply:
Since I'm adding `compile_with`, went ahead and extended this for write support. Should be good to go.